src/main/java/org/commcare/xml/CaseXmlParser.java

```diff
@@ -187,6 +191,10 @@ private void updateCase(Case caseForBlock, String caseId) throws InvalidStructur
                 case "owner_id":
                     String oldUserId = caseForBlock.getUserId();
 
+                    if (oldUserId == null) {
```

Comment:
To fail as early as possible, we should move this error handling to create case block iself after we set userId